Run_2000.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+
+def timer(f: callable, *args) -> float:
+    """
+    A function that receives as a function a function and other parameters.
+    The timer function will measure how long a function f ran when the same
+    parameters are passed to it.
+    :param f: Function
+    :param args: Parameters for the measured function.
+    :return: The running time of f
+    """
+    start = time.time()
+    f(arg for arg in args)
+    return time.time() - start
+
+
+if __name__ == "__main__":
+    pass
```

Comment:
Can you show an example usage?  This piece of code is redundant if you sont include code in it.  Python is not like java, you dont have to have a "main" function, or piece of code